operators/error/Containerfile

```diff
@@ -1,3 +1,3 @@
-FROM interactem/operator
+FROM interactem/interactem-operator
```

Comment:
Not sure I understand this change, we have changed the tag for the base image?

Reply:
This is for clarity when uploading to different org, e.g. samwelborn/operator doesn't really make sense.